config/newrelic.yml

```diff
@@ -21,7 +21,14 @@ production:
   error_collector:
     enabled: true
     capture_source: true
-    ignore_errors: "ActionController::RoutingError,ActionController::BadRequest,ActionDispatch::Http::MimeNegotiation::InvalidType,GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError"
+    ignore_errors: "<%= %w[
+      ActionController::BadRequest
+      ActionController::RoutingError
+      ActionDispatch::Http::MimeNegotiation::InvalidType
+      GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
+    ].tap do |errors|
+      errors << 'Errno::EADDRINUSE' if Identity::Hostdata.instance_role == 'worker'
```

Comment:
It seemed safer to only ignore this on workers, if the web host encountered this error, I would be a little worried? But if we think it would be simpler, I am open to ignoring on all hosts